app/screens/player/player-screen.tsx

```diff
@@ -53,6 +53,10 @@ export const PlayerScreen = observer(() => {
 
   const createAndLoadAndPlay = async () => {
     try {
+      // Make sure audio is played if iOS is in silent mode, defaults to `false`.
+      // NOTE: This sets the property globally which means _all_ future audio
+      // playbacks will be affected.
+      await Audio.setAudioModeAsync({ playsInSilentModeIOS: true })
```

Comment:
I have a funny feeling about setting global options but I don't know why. You definitely have your reasons?

Reply:
I haven't found another way to set this flag. It seems to only be configurable through the global `Audio` instance. 😞